blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,252 @@
+  
+# How to create a Tableau dashboard
+
+**This blog will teach you how to create a Tableau dashboard in the following steps:**
```

Comment:
Can you add more to your introduction to prepare you reader for the blog post? You can explain what is Tableau, why is it important/worthwhile to create a Tableau dashboard, the importance of these basic steps, etc. (You do a good job on doing this towards the end of your blog post.)